projects/aca-folder-rules/assets/i18n/en.json

```diff
@@ -3,6 +3,11 @@
     "ACTIONS": {
       "MANAGE_RULES": "Manage Rules"
     },
+    "TOOLTIPS": {
+      "COLLAPSE_NAVIGATION": "Collapse Navigation",
+      "OPTION_SETTINGS": "Option Settings",
```

Comment:
Can you confirm with the design this is the expected wording? Sounds like "Options" or "Settings" might be a better option

Reply:
As Shane said, it is an options and settings, so I have changed it accordingly.